Dockerfile

```diff
@@ -28,4 +29,5 @@ RUN ARCH=$(dpkg --print-architecture) \
 RUN mkdir -p /etc/nginx/templates
 COPY 40-entrypoint-cloudflared.sh /docker-entrypoint.d/
 RUN chmod +x /docker-entrypoint.d/40-entrypoint-cloudflared.sh
-COPY nginx-default.conf.template /etc/nginx/templates/default.conf.template
\ No newline at end of file
+COPY nginx-default.conf.template /etc/nginx/templates/default.conf.template
+COPY mem_max.conf /etc/sysctl.d/mem_max.conf
```

Comment:
The container doesn't have `sysctl`, it doesn't even have an init process. Why would these be needed here?

Reply:
cloudflared was emitting an error related to the udp buffers available to it; they could get starved. I included the message in commit caada78.

I was trying to apply the adjustments suggested by the error but your point about sysctl is very true. It makes me think the message going away may have just been because privileged true exposes the values it wants vs "fixing" it. I'll chop this one from the PR.

Reply:
this is gone now.